src/libtrx/game/music/common.c

```diff
@@ -13,6 +14,7 @@ static MUSIC_TRACK_ID m_TrackCurrent = MX_INACTIVE;
 static MUSIC_TRACK_ID m_TrackLastPlayed = MX_INACTIVE;
 static MUSIC_TRACK_ID m_TrackDelayed = MX_INACTIVE;
 static MUSIC_TRACK_ID m_TrackLooped = MX_INACTIVE;
+static MUSIC_TRACK_ID m_TrackLastLooped = MX_INACTIVE;
```

Comment:
Do I understand it right that this is used to prevent restarting the ambience from scratch every time Lara crosses a repeatable music trigger? (If so, can we provide a comment in the code?)

Reply:
Yes, exactly that. I've added a comment for this. Thanks.